src/current/_includes/v23.1/prod-deployment/prod-guidance-cache-max-sql-memory.md

```diff
@@ -1 +1,12 @@
-For production deployments, set `--cache` to `25%` or higher. Avoid setting `--cache` and `--max-sql-memory` to a combined value of more than 75% of a machine's total RAM. Doing so increases the risk of memory-related failures.
\ No newline at end of file
+To help guard against OOM events, CockroachDB sets a soft memory limit using mechanisms in Go. Depending on your hardware and workload, you may not need to manually tune `--cache` or `--max-sql-memory`.
```

Comment:
Update details and formula used in Cache and SQL Memory Size section of Recommended Production Settings and Memory section of Common Issues to Monitor.